tests/unit/Codeception/Module/Doctrine2Test.php

```diff
@@ -71,7 +71,12 @@ protected function _setUp()
         require_once $dir . "/CircularRelations/C.php";
         require_once $dir . '/EntityWithUuid.php';
 
-        $connection = DriverManager::getConnection(['driver' => 'sqlite3', 'memory' => true]);
+        $sqliteDriver = 'sqlite3';
+        if (version_compare(InstalledVersions::getVersion('doctrine/orm'), '3.5', '<')) {
```

Comment:
The driver "sqlite3" was added in https://github.com/doctrine/dbal/releases/tag/3.5.0

Reply:
is this change really related to the issue?
Why is this necessary?

Reply:
Yes, it is related since without this change tests would fail if DBAL < 3.5 is installed because the "sqlite3" driver was added in DBAL v3.5.

However, I noticed that I mixed up the package names.
I fixed the version check.